### ISSUE TITLE
[UNBLOCK] Legitimate domains starting with `pixel`, such as `pixelmator.com` and `pixel.google`

### ISSUE THREAD

**Issue by igorburago** [celenityy/bugs#35]
### Which Content Blocker are you using?

uBlock Origin

### With which list(s) does the problem occur?

BadBlock Lite, BadBlock, BadBlock+

### Which domain(s) should be unblocked?

`pixel-sf.com`
`pixel.google`
`pixel.one`
`pixel.withgoogle.com`
`pixelartcss.com`
`pixelhobby.com`
`pixeljoint.com`
`pixelmator.com`
`pixelparmesan.com`
`pixelscan.net`
etc.—the list is obviously not exhaustive.

### Why should the domain(s) be unblocked?

The `||pixel*.*^` and `||pixel.*^` rules in the BadBlock lists are clearly overzealous: there are plenty of legitimate sites which are hosted on or loading assets from domains or subdomains that begin with `pixel`, and which are broken by these rules—see some of the examples listed above.

**Comment by celenityy** [comment]
Hi there, thank you for the report!

I've gone ahead and removed the rules `||pixel*.*^` & `||pixel*.*^$all` - you're right that this was overzealous and unnecessary - not sure what my thinking was there...

These domains should now be unblocked:

```sh
pixel-sf.com
pixelartcss.com
pixelhobby.com
pixeljoint.com
pixelmator.com
pixelparmesan.com
```

However, I think we should keep domains starting with `pixel.*` blocked for now, as this is nearly exclusively used by trackers AFAIK... if this becomes a bigger problem in the future though, I'm open to reconsidering.

I have gone ahead and unblocked:
```sh
pixel.google
pixel.one
pixel.withgoogle.com
```

**Comment by igorburago** [comment]
Agreed. Thank you for a prompt update, and keep up the good work!

**Comment by celenityy** [comment]
Migrated to our new issue tracker at https://codeberg.org/celenity/bugs/issues/35, hope to see you there!
